test/components/EtherscanLink.components.test.tsx

```diff
@@ -30,6 +30,7 @@ describe('<EtherscanLink /> general', () => {
   })
 
   it('does not render when networkId is missing', () => {
+    // @ts-ignore
```

Comment:
Why is this required?

Reply:
does this test even use `networkId` doesn't make sense to me looking at it. NetworkId is resolved in `useWalletConnection` and changing it there shouldnt do anything